Runtime/rendering/editable.cs

```diff
@@ -881,14 +881,14 @@ public class RenderEditable : RenderBox {
 
         public TextPosition getParagraphForward(TextPosition position, TextAffinity? affinity = null) {
             var lineCount = this._textPainter.getLineCount();
-            Paragraph.LineRange line = null;
+            Paragraph.LineRange? line = null;
             for (int i = 0; i < lineCount; ++i) {
                 line = this._textPainter.getLineRange(i);
-                if (!line.hardBreak) {
+                if (!((Paragraph.LineRange) line).hardBreak) {
```

Comment:
how about using line.Value ? just a raw idea, please explain a little if your way is better :-)

Reply:
Okay. That will be better. I just didn't know about the .Value thing when I did this.